prep-html-links/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <div> <a href="../index.html"> click here to go to the contact page </a> </div>
```

Comment:
Hi Derek, this looks like an outdated version of your code. Your video looks correct, but this `href="../index.html"` should link to the page inside of the `contact-us/` directory (_i.e._ `href="contact-us/contact-us.html"`). Any time you make changes to a file, be sure to `git add` it so that it's included in the commit.